package.json

```diff
@@ -22,7 +22,7 @@
     "babel-jest": "^22.4.1",
     "babel-loader": "8.0.0-beta.0",
     "babel-plugin-transform-charcodes": "^0.1.0",
-    "babylon": "7.0.0-beta.46",
+    "@babel/parser": "7.0.0-beta.46",
```

Comment:
We shouldn't be changing the root monorepo packages. There is no `beta.46` of `@babel/parser`.